added info note to cms about mfa on cloud (#5989)
umbraco/UmbracoDocs: 10/umbraco-cms/reference/security/two-factor-authentication.md

Two-factor authentication (2FA) for Umbraco members is activated by implementing an `ITwoFactorProvider` interface and registering the implementation. The implementation can use third-party packages to archive for example support for authentication apps like Microsoft- or Google Authentication App.

{% hint style="info" %}

> If you are using Umbraco Cloud you can use the provided multi-factor authentication feature within the Umbraco ID. You can read more about how to enable it in the [Multi-Factor Authentication](https://docs.umbraco.com/umbraco-cloud/set-up/multi-factor-authentication-on-cloud) article.

Author: ```suggestion
If you are using [Umbraco Cloud](https://umbraco.com/products/umbraco-cloud/) you can use the provided multi-factor authentication feature within the Umbraco ID. You can read more about how to enable it in the [Multi-Factor Authentication](https://docs.umbraco.com/umbraco-cloud/set-up/multi-factor-authentication-on-cloud) article.
```

> If you are using [Umbraco Cloud](https://umbraco.com/products/umbraco-cloud/) you can use the provided multi-factor authentication feature within the Umbraco ID. You can read more about how to enable it in the [Multi-Factor Authentication](https://docs.umbraco.com/umbraco-cloud/set-up/multi-factor-authentication-on-cloud) article.

Reviewer: ```suggestion
If you are using [Umbraco Cloud](https://umbraco.com/products/umbraco-cloud/), you can enable multi-factor authentication in Umbraco ID. For more information, see the [Multi-Factor Authentication](https://docs.umbraco.com/umbraco-cloud/set-up/multi-factor-authentication-on-cloud) article.
```
